pushapkscript/googleplay.py

```diff
@@ -10,6 +10,8 @@
     'dep': 'org.mozilla.fennec_aurora',
 }
 
+_CHANNELS_AUTHORIZED_TO_REACH_GOOGLE_PLAY = ('aurora', 'beta', 'release')
```

Comment:
Question: why is this prefixed with underscore?  It is a public constant, right?

Reply:
I'd like to not expose this function publicily, in favor of `is_allowed_to_push_to_google_play()`

Reply:
Okay, makes sense then! Thanks!